apps/backend/src/tdi-parser/tdi17/TDI17Header.ts

```diff
@@ -0,0 +1,121 @@
+import {
```

Comment:
We could skip the header and trailer as we don't have any use for it  YET. So we can leave it here and use if we need it later.

Reply:
Okay sounds good. I thought we might want the file header as there is the file creation date. So for now I will add an index, and export only the details for 17/34?

Reply:
Yes. 

The date seems to also be in each of the transaction items, so it might just be redundant. 
Is the header date different or providing us any additional context? 🤔